### PR TITLE
Holo-collars, exosuit-wearable collars, and inv overlays

### PR DESCRIPTION
**Pull request by Arokha**
Collars were missing the sprites for showing up on your undersuit icon,
added them.

Holo-collars added. Clicking in-hand lets you set the tag (same code as
labeler, same text length limit).

Made collars fit into exosuit slot as requested by players who want to
wear them while otherwise naked in ERP stuff. LEWD.

### CODE REVIEW

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
 	slot_flags = SLOT_TIE
 	w_class = 2.0
+	icon_override = 'icons/mob/ties.dmi'
 	var/slot = "decor"
 	var/obj/item/clothing/under/has_suit = null		//the suit the tie may be attached to
 	var/image/inv_overlay = null	//overlay used when attached to clothing.
 
 /obj/item/clothing/accessory/New()
 	..()
-	inv_overlay = image("icon" = 'icons/obj/clothing/ties_overlay.dmi', "icon_state" = "[item_color? "[item_color]" : "[icon_state]"]")
```
Comment (Arokha) [inline code/modules/clothing/under/accessories/accessory.dm]: These quotes were bizarre and unnecessary.

File: code/modules/clothing/under/accessories/accessory.dm [hunk 2]
```diff
@@ -215,18 +233,29 @@
 /obj/item/clothing/accessory/collar_pink
 	name = "Pink collar"
 	desc = "This collar will make your pets look FA-BU-LOUS."
+	slot_flags = SLOT_TIE | SLOT_OCLOTHING
 	icon_state = "collar_pnk"
 	item_color = "collar_pnk"
 	item_state = "collar_pnk"
 
-/obj/item/clothing/accessory/scarf
```
Comment (Arokha) [inline code/modules/clothing/under/accessories/accessory.dm]: Moved scarf and tie to be above collars so the proc for holocollar doesn't sit in the middle and break up the flow.

File: code/game/machinery/vending.dm [hunk 3]
```diff
@@ -1146,7 +1146,8 @@
 					/obj/item/clothing/accessory/collar_bell = 3,
 					/obj/item/clothing/accessory/shock_collar = 3,
 					/obj/item/clothing/accessory/collar_spike = 3,
-					/obj/item/clothing/accessory/collar_pink = 3
+					/obj/item/clothing/accessory/collar_pink = 3,
+					/obj/item/clothing/accessory/collar_holo = 3
```
Comment (Arokha) [inline code/game/machinery/vending.dm]: Added holocollar to univend.

**Comment by Leshana** [comment]
These things are all nice

**Pull request merged**